### ISSUE TITLE
VS2022, GitDiffMargin indicator wrongly shows a change, but VS2022 diff margin correctly shows an addition.

### ISSUE THREAD

**Issue by TheGreatUnboxing** [laurentkempe/GitDiffMargin#775]
## Versions
 - Git Diff Margin version: 3.12.1
 - Visual Studio version: Visual Studio Community 2022 (64-bit) - Current Version 17.9.6 (on Windows 10)

## What happened
Have been using git diff margin exclusively for years, however it has often shown changes when there are additions. I thought this might be because of the current diff algorythm being used, so in bash I entered this to make sure:
```
git config --global diff.algorithm patience
```
Nothing changed upon restart, however Sublime Merge does show (correctly) an addition of a block of c# code where Git Diff Margin (incorrectly) shows it as a change.
e.g:

![image](https://github.com/laurentkempe/GitDiffMargin/assets/167497887/dc64c00e-14f4-4482-8e3a-9bc1df46cec8)

![image](https://github.com/laurentkempe/GitDiffMargin/assets/167497887/cc506a1f-4f6b-4ec0-9d7f-814c1a49e755)

I'm wondering if perhaps Git Diff Margin is looking at a different Git install to the one VS is looking at? Or one looking at a 32bit install and the other a 64 bit install? However I don't know how to determine that. My last Git 64bit update installed into C:\Program Files\Git\

### Expected behavior
GitDiffMargin displays the same diff margin indicators as VS 2022 track changes (Git) margin.

**Comment by sharwell** [comment]
For the change above, was there a whitespace change on line 1337?

**Comment by webdevperth** [comment]
> For the change above, was there a whitespace change on line 1337?

No, I did check to make sure there was no whitespace on the blank line there.

> Note that Git Diff Margin is showing _exactly_ the same result as Sublime Merge here. A "change" is defined as one or more lines removed followed by one or more lines added.

True, but both seem incorrect to me, whereas VS 2022 is correct in terms of the actual changes made.

**Comment by TheGreatUnboxing** [comment]
> For the change above, was there a whitespace change on line 1337?

No, I did check to make sure there was no whitespace on the blank line there.

> Note that Git Diff Margin is showing _exactly_ the same result as Sublime Merge here. A "change" is defined as one or more lines removed followed by one or more lines added.

True, but both seem incorrect to me, whereas VS 2022 is correct in terms of the actual changes made.

Ed: As sublime merge agrees with GDM, I was wondering if VS was looking at different diff settings, but tried changing the diff algo in Git and in SM's options, but nothin changed. Perhaps VS is doing some extra black-box magic on top but was hoping to get the same results from GDM (I prefer its margin UI for one)
